src/nyc_trees/apps/home/templates/flatpages/default.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+
+{% block content %}
```

Comment:
Seems like something to address, with design input, when we are focusing on training materials.